.github/workflows/npm.yml

```diff
@@ -18,9 +18,9 @@ jobs:
   npm-compile-publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
           node-version: "14.x"
```

Comment:
Should we update it to 16?

Reply:
Similarily to what I wrote in https://github.com/keep-network/hardhat-helpers/pull/36#discussion_r998185040, we can update the node to v16, but we don't have to - the decision is not related to that GH's deprecation notice...